packages/migrations/src/migrate.ts

```diff
@@ -15,6 +16,7 @@ enum ContractVersions {
     V1 = '1.0.0',
     V2 = '2.0.0',
```

Comment:
Should we just remove `V2` here? It seems like we should be using either `V2Testnet` or `V2Mainnet` everywhere.